packages/web-app-files/src/helpers/resources.ts

```diff
@@ -384,7 +385,7 @@ export function buildSharedResource(
     }
     resource.canDownload = () => parseInt(share.state) === ShareStatus.accepted
     resource.canShare = () => SharePermissions.share.enabled(share.permissions)
-    resource.canRename = () => SharePermissions.update.enabled(share.permissions)
+    resource.canRename = () => true
```

Comment:
@kulmann I don't know if we can leave it this way, or we need a distinction between ocis and oc10